frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -27,8 +28,20 @@ const waiveFeeRelationship = [
   }
 ];
 
+const employerInputs = [
```

Comment:
Why an array of objects instead of just an array of strings? From what I can tell, the empty string values are being completely ignored inside of TextInputGroup.

Reply:
I am using an array of objects here as the key is the input label and the value is the the actual value of the input.

Reply:
But the user of the TextInputGroup component does not provide values for the input. The values come from the context/dataKey, and always start off as empty.

I still don't think it makes sense for the user of TextInputGroup to pass in an object. It seems like an internal implementation detail of TextInputGroup that is leaking out to the consumers of the component.

Reply:
Sorry for the late response. I am using the object to group inputs. I have been trying to refactor and I am having a hard time understanding how to add and remove groups of inputs.

Reply:
Why do the inputs need to be grouped? The example here only has one object inside of the array. Are there going to be cases where there will be two objects inside of the array?

Reply:
Yes when you add a group it adds another object of group inputs.

Reply:
Do we have a use case for that?

If not, then let's not add it. The motion to waive fees form doesn't seem to have it as far as I can tell?

If it does, then I would propose the following:

```js
[
  ['firstName', 'lastName']
  ['income', 'profession']
]
```

Reply:
It's the empty strings that don't need to be there that are bothering me.

Reply:
Okay that makes more sense. I will get that done.